src/main/java/com/lothrazar/storagenetwork/gui/NetworkWidget.java

```diff
@@ -380,4 +385,13 @@ public void render() {
       jeiBtn.setTextureId(gui.isJeiSearchSynced() ? TextureEnum.JEI_GREEN : TextureEnum.JEI_RED);
     }
   }
+
+  public interface ISearchHandler {
```

Comment:
aha, search handler interface so that both jei and emi can hook into this. i see good solution for the problem yes